egs2/css10/tts1/conf/mfcc.conf

```diff
@@ -0,0 +1,7 @@
+--sample-frequency=16000
+--frame-length=25 # the default is 25
+--low-freq=20 # the default.
+--high-freq=7600 # the default is zero meaning use the Nyquist (8k in this case).
+--num-mel-bins=30
+--num-ceps=30
+--snip-edges=false
```

Comment:
Is this file required for this recipe?

Reply:
This is need if we want to extract x-vector.

Reply:
I see. Thanks.